tests/acceptance/builds/pull-requests-test.js

```diff
@@ -0,0 +1,81 @@
+import { test } from 'qunit';
```

Comment:
I guess this comes out weird in the review diff. Really just 👍 on adding another test.